extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -33,6 +31,7 @@
 
 	&__counter {
 		color: $color-text;
+		padding-bottom: 12px;
```

Comment:
I wonder if it's possible to have counter (or language switcher) without buttons - because than we would have too big gap between counter/langswitcher and the line separator. Maybe it would be safer to do something like:
`.counter + .buttons { margin-top: 12px; }
.languages + .counter { margin-top: 12px; }
.languages + .buttons { margin-top: 12px; }`